wger/core/templates/template_features.html

```diff
@@ -75,54 +82,41 @@
 
 
 <footer class="bg-darkest-gray">
-    <div class="container">
-        <div class="row">
-            <div class="col-md-4">
-                <ul class="list-inline quicklinks">
-                    <li><a href="{% url 'software:contribute' %}">{% trans "Contribute!" %}</a></li>
-                    <li><a href="{% url 'core:contact' %}">{% trans "Get in touch" %}</a></li>
+    <div class="row">
+        <div class="col-lg-2" id="footer-logo">Wger</div>
+        <div class="col-lg-6" id="footer-nav">
+            <div class="col-lg-4">
+                <ul>
+                    <li class="footer-nav-header"> Product</li>
+                    <li><a href="{% url 'core:about' %}">About</a></li>
+                    <li><a href="{% url 'software:tos' %}">{% trans "Terms & Service"%}</a></li>
+                    <li><a href="https://wger.de/en/software/issues">Support</a></li>
```

Comment:
Exactly, it is a mistake that we had made, we forgot to check out the branch and made those changes